transformer_lens/loading_from_pretrained.py

```diff
@@ -138,6 +138,22 @@
     "stabilityai/stablelm-tuned-alpha-7b",
     "bigscience/bloom-560m",
     "bigcode/santacoder",
+    "Qwen/Qwen-1_8B",
+    "Qwen/Qwen-7B",
+    "Qwen/Qwen-14B",
+    # "Qwen/Qwen-72B",
+    "Qwen/Qwen-1_8B-Chat",
+    "Qwen/Qwen-7B-Chat",
+    "Qwen/Qwen-14B-Chat",
+    # "Qwen/Qwen-72B-Chat",
+    # "Qwen/Qwen-1_8B-Chat-Int4",
```

Comment:
nit: can we clean up these quantized versions

Reply:
done